admin/class-yoast-plugin-conflict.php

```diff
@@ -180,7 +180,7 @@ protected function set_error( $plugin_section, $readable_plugin_section ) {
 			new Yoast_Notification(
 				$error_message,
 				array(
-					'type'      => 'error yoast-dismissible',
+					'type'      => 'error',
```

Comment:
I think it is cool to have a class constant in `Yoast_Notification` containing the `error` value. It makes searching a lot easier and it restricts the notification types a little bit.